src/app/pages/login/login.page.html

```diff
@@ -6,16 +6,21 @@
     <div padding class="span">
       <ion-img src="./assets/imgs/loginlogo.jpg"></ion-img>
       <br>
+      <!-- Error messages -->
+      <span class="error ion-padding" *ngIf="formErrors?.signIn">
+        <ion-icon name="alert" color="danger"></ion-icon>The mobile number, email, or password was incorrect.
```

Comment:
An error on the form group level alerting to AngularFire2/Firebase sign-in failures.